### PR TITLE
chore: remove enterprise packages dependencies

### PR DESCRIPTION
**Pull request by lukasborawski**
### Description

---

Getting rid of the enterprise packages.

### Type of change

---

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

---

_none_

### Screenshots

---

_none_

### Checklist:

---

- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: apps/web/pages/category.tsx [hunk 1]
```diff
@@ -39,7 +46,11 @@ export default function CategoryPage() {
   }
 
   const { products, pagination, subCategories, facets } = productsCatalog;
-  const categories = subCategories.map(({ name, productCount }) => ({ name, count: productCount, href: '/category' }));
+  const categories: CategoryTreeItem[] = subCategories.map(({ name, productCount }) => ({
+    name,
+    count: productCount || 0,
```
Comment (dawid-ziobro) [inline apps/web/pages/category.tsx:51]: I've had exactly the same issue for the other BP. I ended with `count: productCount || undefined` as `0` is still valid number for us.

File: apps/web/package.json [hunk 2]
```diff
@@ -46,16 +48,13 @@
...
-    "cross-env": "^7.0.3",
-    "cypress": "^12.12.0",
     "eslint-config-next": "^13.3.0",
-    "jest": "^29.5.0",
     "jest-environment-jsdom": "^29.2.2",
     "postcss": "^8.4.23",
     "sass": "^1.62.0",
+    "jest": "^29.5.0",
     "tailwindcss": "^3.3.1",
-    "ts-jest": "^29.0.3"
```
Comment (dawid-ziobro) [inline apps/web/package.json:59]: I believe `ts-jest` is still needed.

File: apps/web/package.json [hunk 3]
```diff
@@ -17,9 +18,12 @@
     "@storefront-ui/react": "~2.2.0",
     "@storefront-ui/shared": "~2.2.0",
     "@tanstack/react-query": "^4.29.5",
-    "@vsf-enterprise/sdk": "^1.0.0",
-    "@vsf-enterprise/storefront-boilerplate-sdk": "0.1.0-rc.2",
+    "@tanstack/react-query-devtools": "^4.29.5",
+    "@vue-storefront/sdk": "~1.0.1",
+    "@vue-storefront/storefront-boilerplate-sdk": "~0.1.0",
+    "@vue-storefront/unified-data-model": "~0.1.0",
```
Comment (dawid-ziobro) [inline apps/web/package.json:24]: Why to dependencies? This package is only serving types.

File: apps/web/package.json [hunk 4]
```diff
@@ -17,9 +18,12 @@
     "@storefront-ui/react": "~2.2.0",
     "@storefront-ui/shared": "~2.2.0",
     "@tanstack/react-query": "^4.29.5",
-    "@vsf-enterprise/sdk": "^1.0.0",
-    "@vsf-enterprise/storefront-boilerplate-sdk": "0.1.0-rc.2",
+    "@tanstack/react-query-devtools": "^4.29.5",
```
Comment (dawid-ziobro) [inline apps/web/package.json:21]: Why moved to deps? It is not used for production.

File: apps/web/components/ui/Review/Review.tsx [hunk 5]
```diff
@@ -9,16 +9,16 @@ export function Review({ review, ...attributes }: ReviewProps) {
 
   const [isCollapsed, setIsCollapsed] = useState(true);
   const charLimit = 400;
-  const isButtonVisible = text?.length > charLimit;
-  const truncatedContent = isButtonVisible && isCollapsed ? `${text.slice(0, charLimit)}...` : text;
+  const isButtonVisible = text?.length! > charLimit;
```
Comment (dawid-ziobro) [inline apps/web/components/ui/Review/Review.tsx:12]: Seems that more types have changed...

File: apps/web/package.json (hunk 3 above)
Comment (lukasborawski) [reply apps/web/package.json:24]: Thanks correct, massive copying. Thanks.

File: apps/web/components/ui/Review/Review.tsx (hunk 5 above)
Comment (lukasborawski) [reply apps/web/components/ui/Review/Review.tsx]: Fixed.

**Review by marcin-saja — APPROVED** [review]

File: apps/web/package.json (hunk 4 above)
Comment (dawid-ziobro) [reply apps/web/package.json:21]: It's about `"@tanstack/react-query-devtools"`

Comment (lukasborawski) [reply apps/web/package.json:21]: Right, I think it was here from the beginning... Will move it.

File: apps/web/.lintstagedrc.js
Comment (Razz21) [inline apps/web/.lintstagedrc.js:1]: Not sure about this setup, but if it works as expected than its fine

File: apps/web/components/CartPageContent/CartPageContent.tsx [hunk 6]
```diff
@@ -20,8 +20,8 @@ export function CartPageContent() {
             imageUrl={item.image?.url}
             imageAlt={item.image?.alt}
             name={item.name}
-            price={item.totalPrice.amount}
-            specialPrice={item.unitPrice?.value?.amount}
+            price={item.totalPrice?.amount || 0}
+            specialPrice={item.unitPrice?.value?.amount || 0}
```
Comment (Razz21) [inline apps/web/components/CartPageContent/CartPageContent.tsx:24]: I believe `unitPrice` will be displayed as discount price there

**Review by Razz21 — APPROVED** [review]

File: .github/workflows/run-lhci-test.yml [hunk 7]
```diff
@@ -1,6 +1,4 @@
-# For now this workflow is manually disabled in GH Actions, 
-# enable it again when sdk mock package will be released and build will be working on GH
-name: LHCI Test
+name: Testing
```
Comment (dawid-ziobro) [inline .github/workflows/run-lhci-test.yml:1]: Doubled workflow name.

File: .github/workflows/conventional-pr-name.yml [hunk 8]
```diff
@@ -1,4 +1,4 @@
-name: Commitlint
+name: Linting
```
Comment (dawid-ziobro) [inline .github/workflows/conventional-pr-name.yml:1]: Doubled workflow name.

Comment (lukasborawski) [reply .github/workflows/conventional-pr-name.yml:1]: @dawid-ziobro it's intentional, this is the global naming to be connected with each other.

Zrzut ekranu 2023-07-6 o 12 31 44

**Review by lukaszjedrasik — APPROVED** [review]

**Review by dawid-ziobro — APPROVED** [review]

**Pull request merged**